extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/local_inference/text_generation.py

```diff
@@ -283,7 +296,14 @@ def get_output_text(
         # TODO (rossdanlm): Handle multiple outputs in list
         # https://github.com/lastmile-ai/aiconfig/issues/467
         if output.output_type == "execute_result":
-            if isinstance(output.data, str):
-                return output.data
-        else:
-            return ""
+            output_data = output.data
+            if isinstance(output_data, str):
+                return output_data
+            if isinstance(output_data, OutputDataWithValue):
+                if isinstance(output_data.value, str):
+                    return output_data.value
+                # HuggingFace Text generation does not support function
+                # calls so shouldn't get here, but just being safe
+                return json.dumps(output_data.value, indent=2)
+            raise ValueError("Not Implemented")
```

Comment:
For these cases we could technically json.dumps(output_data), right?

Reply:
I should not ever be possible to get to this point, which is why I raised an error to be cautious. But yea I can just remove error and fall into the final `return ""` instead